tests/benches/coprocessor_executors/mod.rs

```diff
@@ -26,7 +26,7 @@ fn execute<M: criterion::measurement::Measurement + 'static>(c: &mut criterion::
     c.final_summary();
 }
 
-#[cfg(target_os = "linux")]
+#[cfg(all(target_os = "linux", target_arch = "x86_64"))]
```

Comment:
Please also update the `not` part on line 55

Reply:
Done. Good catch.